context/app/static/js/components/Detail/EntityTile/EntityTile.jsx

```diff
@@ -50,6 +56,18 @@ function EntityTile(props) {
               )}
             </TextSection>
           </FixedWidthFlex>
+          <BottomSection $isCurrentEntity={isCurrentEntity}>
+            {Object.entries(descendantCounts).map(([k, v]) => (
+              <>
+                {k === 'Dataset' ? <BottomDatasetIcon /> : <BottomSampleIcon />}
```

Comment:
```suggestion
                {k === 'Dataset' && <BottomDatasetIcon />}
                {k === 'Sample' && <BottomSampleIcon />}
```
Same commend in a related PR?